mantidproject/mantid
# ISIS Powder - Support partial run selection

Description of work.
This PR adds the ability to specify a file extension allowing users to pick between partial (`.s0`) files.

**To test:**
Code review

- Download this .zip from [here](https://ufile.io/o46v7)
- Extract it to a known location
- Add the `Input Data` folder within it to the list of input directories in Mantid (File -> Manage User Directories)
- Open the script in Mantid
- There is a line saying `Enter path to extracted files here`. Enter the path to the extracted folder there
- Run the script
- You should have two output workspaces with `.s1` and `.s2` differencing between them in Mantid
- Check the output folder which will have been created in the extracted folder. You should see something similar. 

Fixes #19500 .

*Does not need to be in the release notes. - Scripts still under development*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
